### PR TITLE
Update origins to Council

### PR DESCRIPTION
**Pull request by joepetrowski**
Addresses https://github.com/paritytech/statemint/pull/50#discussion_r630958254

### CODE REVIEW

File: runtime/statemine/src/lib.rs [hunk 1]
```diff
@@ -258,7 +258,7 @@ parameter_types! {
 	// https://github.com/paritytech/substrate/blob/069917b/frame/assets/src/lib.rs#L257L271
 	pub const MetadataDepositBase: Balance = deposit(1, 68);
 	pub const MetadataDepositPerByte: Balance = deposit(0, 1);
-	pub const UnitBody: BodyId = BodyId::Unit;
+	pub const UnitBody: BodyId = BodyId::Executive;
```
Comment (riusricardo) [inline runtime/statemine/src/lib.rs:261]:
```suggestion
	pub const ExecutiveBody: BodyId = BodyId::Executive;
```

File: runtime/statemint/src/lib.rs [hunk 2]
```diff
@@ -259,7 +259,7 @@ parameter_types! {
 	// https://github.com/paritytech/substrate/blob/069917b/frame/assets/src/lib.rs#L257L271
 	pub const MetadataDepositBase: Balance = deposit(1, 68);
 	pub const MetadataDepositPerByte: Balance = deposit(0, 1);
-	pub const UnitBody: BodyId = BodyId::Unit;
+	pub const UnitBody: BodyId = BodyId::Executive;
```
Comment (riusricardo) [inline runtime/statemint/src/lib.rs:262]:
```suggestion
	pub const ExecutiveBody: BodyId = BodyId::Executive;
```

**Review by riusricardo — COMMENTED** [review]
The name should be changed with the respective changes:
https://github.com/paritytech/statemint/blob/master/runtime/statemint/src/lib.rs#L269
https://github.com/paritytech/statemint/blob/master/runtime/statemint/src/lib.rs#L616
https://github.com/paritytech/statemint/blob/master/runtime/statemine/src/lib.rs#L268
https://github.com/paritytech/statemint/blob/master/runtime/statemine/src/lib.rs#L615

File: runtime/statemine/src/lib.rs [hunk 3]
```diff
@@ -500,17 +500,17 @@ parameter_types! {
...
 	};
 }
 
 pub type Barrier = (
 	TakeWeightCredit,
 	AllowTopLevelPaidExecutionFrom<All<MultiLocation>>,
-	AllowUnpaidExecutionFrom<ParentOrParentsUnitPlurality>,
-	// ^^^ Parent & its unit plurality gets free execution
+	AllowUnpaidExecutionFrom<ParentOrParentsExecutivePlurality>,
+	// ^^^ Parent and its exec plurality get free execution
```
Comment (apopiak) [inline runtime/statemine/src/lib.rs:513]: Maybe out of scope for this PR, but maybe we should just give any parent Plurality free execution.

**Review by apopiak — APPROVED** [review]

**Pull request merged**